code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -2730,6 +2742,17 @@
 	reagents.add_reagent(NUTRIMENT, 7)
 	bitesize = 2
 
+/obj/item/weapon/reagent_containers/food/snacks/polypwich
+	name = "Polypwich"
+	desc = "Polyp meat and gelatin between two slices of bread makes for a nutritious sandwich. Unfortunately it has a soggy and unpleasant texture. These are commonly served to mothership prisoners who misbehave."
```

Comment:
Could use a small chance to make you get a nausea message or something. "The texture almost makes you vomit!"

Reply:
Added one.